CHANGELOG.md

```diff
@@ -1,3 +1,146 @@
+# 0.0.117 - XXX - "Everything but the Twelve Sinks"
+
+## API Updates
+ * `ProbabilisticScorer`'s internal models have been substantially improved,
+   including better decaying (#1789), a more granular historical channel
+   liquidity tracker (#2176) and a now-default option to make our estimate for a
+   channel's current liquidity nonlinear in the channel's capacity (#2547). In
+   total, these changes should result in improved payment success rates at the
+   cost of slightly worse routefinding performance.
+ * Support for custom TLVs for recipients of HTLCs has been added (#2308).
+ * Support for generating transactions for third-party watchtowers has been
```

Comment:
It can be noted justice transactions claiming only the revoked `to_local`, not potential revoked HTLC output or second-stage HTLC outputs, iirc #2337 changes.